job-dsls/jobs/prod/prod_rhbop_trigger_umb_message.groovy

```diff
@@ -14,8 +14,8 @@ pipelineJob("${folderPath}/rhbop-trigger-umb-message") {
         stringParam('PRODUCT_ID', '', 'The product ID. This parameter is optional and in case it is not defined, the default OSL product ID is used.')
         stringParam('MILESTONE', '', 'The release milestone, i.e. 8.29.0.CR1. This parameter is optional and in case it is not defined, the latest available milestone is used.')
         stringParam('QUARKUS_VERSION', '2.7.6.Final-redhat-00006', 'The productized version of Quarkus used for building Red Hat build of OptaPlanner currect milestone.')
-        stringParam("PNC_API_URL", "http://orch.psi.redhat.com/pnc-rest/v2", "PNC Rest API endpoint. See: https://docs.engineering.redhat.com/display/JP/User%27s+guide")
-        stringParam("INDY_URL", "http://indy.psi.redhat.com/browse/maven/hosted/pnc-builds/", "Indy PNC builds repository")
+        stringParam("PNC_API_URL", "\${env.ORCH_PSI_URL}/pnc-rest/v2", "PNC Rest API endpoint. See: \${env.DOCS_ENGINEERING_URL}/display/JP/User%27s+guide")
+        stringParam("INDY_URL", "\${env.INDY_PNC_URL}", "Indy PNC builds repository")
```

Comment:
env variables can't be used here, please move this value inside the job itself

Reply:
according to https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/custom/job/emingora/job/envtest/2/console it can be used, so I agree on leave it as it is.